portfolio/src/main/java/com/google/sps/usercomment/UserComments.java

```diff
@@ -22,15 +22,19 @@
   private String comment;
   private String email;
   private String userId;
+  private double commentSentimentScore;
   private long id;
 
-  /** Constructor(String, String, long)*/
-  public UserComments(String userInput, String commentInput, String userEmail, String userID, long entityID) {
-    user = userInput;
-    comment = commentInput;
-    email = userEmail;
-    userId = userID;
-    id = entityID;
+  public UserComments(
+    String user, String comment, String email, 
+    String userId, double commentSentimentScore, long id) {
+
+    this.user = user;
+    this.comment = comment;
+    this.email = email;
+    this.userId = userId;
+    this.commentSentimentScore = commentSentimentScore;
```

Comment:
```suggestion
    this.sentimentScore = sentimentScore;
```
Nit: You usually don't have to repeat the word "comment" if you are inside the comment class. See below for a "bad" example:

```
public class Dog {
    int dogAge;
    string dogName;
    string dogBreed;
}
```

Here's the "good" example:

```
public class Dog {
    int age;
    string name;
    string breed;
}
```
Does that make sense?

Reply:
It surely does! Thank you Aki.